src/Dfe.EarlyYearsQualification.Web/Controllers/CheckAdditionalRequirementsController.cs

```diff
@@ -20,35 +20,54 @@ public class CheckAdditionalRequirementsController(
     IUserJourneyCookieService userJourneyCookieService)
     : ServiceController
 {
-    [HttpGet("{qualificationId}")]
-    public async Task<IActionResult> Index(string qualificationId)
+    [HttpGet("{qualificationId}/{questionId}")]
+    public async Task<IActionResult> Index(string qualificationId, int questionId)
```

Comment:
Suggestion (minor): it's not a question ID. Would `questionIndex` be a better name for this param (same on other action methods too)?

Reply:
Changed all references to questionIndex, definitely a better name :-)